crates/coin-store/src/store.rs

```diff
@@ -13,6 +13,7 @@ use crate::error::StoreError;
 use crate::filter::Filter;
 
 static MIGRATOR: Migrator = sqlx::migrate!();
+const BLINDING_KEY_LEN: usize = 32;
```

Comment:
Let's have a single constant, look whether elements already have it (related to the SEED_LEN)